handler/core/strategy.go

```diff
@@ -0,0 +1,22 @@
+package core
+
+import (
+	"github.com/ory-am/fosite"
+	"golang.org/x/net/context"
+	"net/http"
+)
+
+type AccessTokenStrategy interface {
+	GenerateAccessToken(ctx context.Context, req *http.Request, requester fosite.AuthorizeRequester, session interface{}) (token string, signature string, err error)
```

Comment:
I'm not sure if it is necessary or even desirable to pass the context as it would right now only offer capabilities for aborting the request, which does not really make sense because we're not waiting for a response or anything